web/stories/historia-prensa.rst

```diff
@@ -18,16 +18,23 @@ Año 2018
 
    <div class="row">
 
+.. template:: prensa-image
+   :title: PyCon Japón
+   :description: PyCon JP 2018: Manuel Kaufmann- keynote speaker on the first day
```

Comment:
La descripción debería ser:

PyCon JP 2018: About the keynote speaker on the first day